cmd/oras/manifest/push.go

```diff
@@ -35,9 +37,11 @@ type pushOptions struct {
 	option.Pretty
 	option.Remote
 
-	targetRef string
-	fileRef   string
-	mediaType string
+	concurrency int64
+	targetRef   string
+	extraRefs   []string
+	fileRef     string
+	mediaType   string
 }
 
 func pushCmd() *cobra.Command {
```

Comment:
The `Use` of `cobra.Command` should also be updated after #596.

Reply:
done